python/lsst/ap/verify/dataset.py

```diff
@@ -277,7 +277,10 @@ def makeCompatibleRepoGen3(self, repoDir):
         """
         # No way to tell makeRepo "create only what's missing"
         try:
-            repoConfig = dafButler.Butler.makeRepo(repoDir)
+            initConfig = dafButler.Config()
```

Comment:
This doesn't matter but in `butler create` command line we call this a seed config.